docs/extras/permissions/permissions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Permissions
+sidebar_position: 1
+---
+
+# Permissions
+
+At DevCycle, our primary goal for permissions is to protect production. We've designed our permission to be lightweight, straightforward and simple while ensuring production is protected.
+
+Specifically, we offer Permissions Gating to protect Environments which are marked as "production" from unauthorized changes. This allows Organizations to manage which users can modify the Targeting Status, Rules and served Variations in a production Environment that may impact their end users.
+
+Permissions can be enabled for Organizations on our Business or Enterprise plans. Please contact our [support](mailto:support@devcycle.com) team to get started.
+
+:::info
+Permissions enforcement will apply to all Projects and Production-type Environments within your Organization.
+:::
+
+## What Can Users Do?
+
+To ensure usability, all users are able to make any edits to a Feature when production targeting isn't enabled. Permissions Gating and it's protections come into play when production targeting is either enabled or a user is attempting to enable production.
+
+See below for the full details on what each user role can do when permissions are enabled.
+
+### Members
+
+- ✅ Configure Development and Staging Environments
+- ✅ Configure **Inactive** Production Environments
+- ❌ Configure **Active** Production Environments, Variables and Variations
```

Comment:
@kaushalkapasi can you split variables and variations out into their own line and write
"When a production environment's targeting is enabled, all editing of Variables and Variations is disabled for members

Reply:
Going to use:
`- ❌ When Production is **Active**, Configure Variables and Variations`